radio/src/targets/common/arm/stm32/diskio_sdio.cpp

```diff
@@ -118,6 +115,24 @@
 #  define SD_SDIO_HARDWARE_FLOW_CONTROL_DISABLE SDIO_HARDWARE_FLOW_CONTROL_DISABLE
 #endif
 
+struct {
+  SD_SDIO_TypeDef* periph = SD_SDIO;
+  gpio_t  D0 = SD_SDIO_PIN_D0;
+  gpio_t  D1 = SD_SDIO_PIN_D1;
+  gpio_t  D2 = SD_SDIO_PIN_D2;
+  gpio_t  D3 = SD_SDIO_PIN_D3;
+  gpio_t  CMD = SD_SDIO_PIN_CMD;
+  gpio_t  CLK = SD_SDIO_PIN_CLK;
+  gpio_af_t  AF_D0 = SD_SDIO_AF_D0;
+  gpio_af_t  AF_D1 = SD_SDIO_AF_D1;
+  gpio_af_t  AF_D2 = SD_SDIO_AF_D2;
+  gpio_af_t  AF_D3 = SD_SDIO_AF_D3;
+  gpio_af_t  AF_CMD = SD_SDIO_AF_CMD;
+  gpio_af_t  AF_CLK = SD_SDIO_AF_CLK;
+  IRQn_Type  IRQn = SD_SDIO_IRQn;
+  uint32_t SD_CLK_DIV = SD_SDIO_TRANSFER_CLK_DIV;
+} currentSD;
```

Comment:
It would more effective in terms of flash usage to use `const` to define the actual data structure, an then have `currentSD` be `const struct *`.